operator_ui/__tests__/connectors/reducers/jobRuns.test.js

```diff
@@ -93,4 +94,24 @@ describe('connectors/reducers/jobRuns', () => {
       b: { id: 'b' }
     })
   })
+  it('RECEIVE_DELETE_SUCCESS deletes jobrun associations', () => {
```

Comment:
✂️ this no longer exists

Reply:
@rupurt Not sure what you mean. Can you elaborate on this one?

Reply:
`runs` should no longer be returned from the jobs#show API endpoint